src/Api/Service/Sf.php

```diff
@@ -959,4 +991,59 @@ public function update_last_activity($newtime = null)
         }
         $this->app['session']->set('last_activity', $newtime);
     }
+
+    // -------------------------------- Review & Suggest Api ----------------------------------
+
+    public function rs_get_words($numWords = 64){
```

Comment:
The namespace of **rs_** carries very little semantic content; **suggest_** would be better. It might be meaningful to you today because you have been using this abbreviation but not to others trying to understand your code. I realize we already have that problem with DTO (see my comment above) but we don't want to introduce new problems :).